README.md

```diff
@@ -86,6 +86,9 @@ paginate = 5
   centerTheme = false
   # set a custom favicon (default is a `themeColor` square)
   # favicon = "favicon.ico"
+  # set post to show the last updated
+  # If you use git, you can set `enableGitInfo` to `true` and then post will automatically get the last updated
+  showLastUpdated = true
```

Comment:
If we want to have this `false` by default, you should change the value to `false` or comment this line though.

Reply:
I will change the value to `false` :-D